.Lib9c.Tests/Action/DailyRewardTest.cs

```diff
@@ -79,6 +79,9 @@ public void Rehearsal()
             var updatedAddresses = new List<Address>
             {
                 _avatarAddress,
+                _avatarAddress.Derive(LegacyInventoryKey),
+                _avatarAddress.Derive(LegacyWorldInformationKey),
+                _avatarAddress.Derive(LegacyQuestListKey),
```

Comment:
IMO, it would be better if we have the assertion about re-creating from output states in `Execute()` test case as regression.